user/pb/user.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+option go_package = "./user/pb";
+
+service UserService {
+    rpc Authenticate(UserAuthRequest) returns (UserAuthResponse) {}
+    rpc CreateUser(CreateUserRequest) returns (CreateUserResponse) {}
+    rpc UpdateUser(UpdateUserRequest) returns (UpdateUserResponse) {}
+    rpc GetUser(GetUserRequest) returns (GetUserResponse) {}
+    rpc DeleteUser(DeleteUserRequest) returns (DeleteUserResponse) {}
+}
+
+message UserAuthRequest {
+    string username = 1;
+    string password = 2;
+}
+
+message UserAuthResponse {
+    string message = 1;
+}
+
+message CreateUserRequest {
+    string name = 1;
+    string password = 2;
+    string age = 3;
+    string additional_information = 4;
```

Comment:
snake case is not go complaint, run go fmt, and please pay attention to the warning in the compilation stage